aspnetcore/blazor/security/webassembly/index.md

```diff
@@ -20,10 +20,10 @@ The Blazor WebAssembly security documentation primarily focuses on how to accomp
 
 A Blazor WebAssembly app's .NET/C# codebase is served to clients, and the app's code can't be protected from inspection and tampering by users. Never place sensitive data into a Blazor WebAssembly app, such as app secrets, connection strings, passwords, security keys, and private .NET/C# code.
 
-The following technologies are useful for storing sensitive data, which can be used together in the same app to split responsibilities for storing data between Development and Staging/Production environments:
+The following technologies are useful for storing sensitive data, which can be used together in the same app to split responsibilities for storing data between Development and `Staging`/`Production` environments:
```

Comment:
Consider this minor change:
```suggestion
The following technologies are useful for storing sensitive data, which can be used together in the same app to split responsibilities for storing data between `Development`, `Staging` and `Production` environments:
```

Reply:
Also, it should be "among."

Reply:
I'm on my crappy phone, so its hard to adjust here.

Ill try. 😆